server/data/data-jmap/src/main/java/org/apache/james/jmap/api/preview/Preview.java

```diff
@@ -30,19 +30,17 @@
 public class Preview {
 
     private static final int MAX_LENGTH = 256;
-    public static final Preview NO_BODY = new Preview("(Empty)");
 
     public static Preview from(String value) {
         return new Preview(value);
     }
 
-    public static Preview compute(String textBody) {
+    public static Optional<Preview> compute(String textBody) {
         return Optional.of(textBody)
             .map(StringUtils::normalizeSpace)
             .filter(text -> !text.isEmpty())
```

Comment:
If removing this step, and return a Preview containing an "" string, it then still be stored in the database, I rather return an Optional<Preview> and let the caller not to store the preview in case of empty optional.